programs/server/config.xml

```diff
@@ -202,6 +202,10 @@
       -->
     <interserver_http_port>9009</interserver_http_port>
 
+    <settings>
+        <log_query_settings>1</log_query_settings>
```

Comment:
This is not the correct place to modify settings for the tests, please modify ClickHouse/tests/configs/ instead

Reply:
Addressed the review comments